### PR TITLE
Add: animal info

### PR DESCRIPTION
**Pull request by maysunktong**
An Object of animal info

### CODE REVIEW

File: animal-info.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      food: ["kangaroos", "rabbits", "lizards", "birds"],
+      description:
+        "The perentie (Varanus giganteus) is the largest monitor lizard or goanna native to Australia. It is one of the largest living lizards on earth, after the Komodo dragon, Asian water monitor, crocodile monitor, and intersecting by size with Nile monitor. Found west of the Great Dividing Range in the arid areas of Australia, it is rarely seen, because of its shyness and the remoteness of much of its range from human habitation. The species is considered to be a least-concern species according to the International Union for Conservation of Nature. Its status in many Aboriginal cultures is evident in the totemic relationships, and part of the Ngiṉṯaka dreaming, as well as bush tucker. It was a favoured food item among desert Aboriginal tribes, and the fat was used for medicinal and ceremonial purposes.",
+      length: "2.5m",
+      weight: "20kg",
+      found: "Deserts",
+    },
+  },
+  mammals: {
+    echidna: {
```
Comment (Endorze) [inline animal-info.js]: You have an extra open bracket without a corresponding closing bracket. This mismatch will cause a syntax error and prevent the code from running correctly when we try to use this list of objects, remove and resubmit. Line 36. //Edit the code was fine, i missread it.

**Review by Endorze — CHANGES_REQUESTED** [review]
Fix the error on line 36 to avoid syntax error, otherwise the code looks amazing!

**Review by Endorze — APPROVED** [review]
Looks fine!

**Comment by maysunktong** [comment]
I have add few more animals to be 6 animals of each category. Also, put each data in a more suitable type.

**Review by Endorze — APPROVED** [review]
Looks great!

**Comment by maysunktong** [comment]
Change lifespan, length, weight back to [string].

**Review by mvic89 — COMMENTED** [review]
Looks good! nothing seems to be out of order.

**Review by mvic89 — APPROVED** [review]

**Pull request merged**